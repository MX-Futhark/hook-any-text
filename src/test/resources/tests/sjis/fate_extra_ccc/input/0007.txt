82B182CC82A082BD82E882AA814182D982E781420A95978B4988CF88F582C682B582C4927082B882A982B582A282E682CB81480A82C582E0816381630063816300000000000100000002000000030000000400000005000000060000000700000008000000090000000A0000000B0000000C0000000D0000000E0000000F000000100000001100000012000000130000001400000015000000160000001700000018000000190000001A0000001B0000001C000002000000000000000000000000000000000000001700C201000000002818A3480B0000000000000000000000000000000000000002001000000000000000000000000000FFFFFFDC0000000000003C43000000000000F04300000000FFFFFFDC0000F04300003C4300000000000000000000A842FFFFFFDC0000000000008843000000000000F0430000A842FFFFFFDC0000F0430000884300000000FFFFFFDC0B0000000000000000000000000000000000000003001000000000000000000000000000FFFFFFDC000060C100001F43000000000000184300000000FFFFFFDC00000A4300001F43000000000000000000001843FFFFFFDC000060C100809B43000000000000184300001843FFFFFFDC00000A4300809B4300000000FFFFFFDC0B000000AC15C90800000000000000000000000004001000000100000000000000000000FFFFFFDC000000C000002A43000000000000004300000000FFFFFFDC0000FC4200002A43000000000000000000000043FFFFFFDC000000C000009543000000000000004300000043FFFFFFDC0000FC420000954300000000FFFFFFDC0B0000000000000000000000000000000000000005001000000100000000000000000000FFFFFFDC0000604100003A43000000000000C04200000000FFFFFFDC0000DC4200003A4300000000000000000000C042FFFFFFDC0000604100008D43000000000000C0420000C042FFFFFFDC0000DC4200008D4300000000FFFFFFDC0B000000AC16C9080000000000000000000000000700100000000000