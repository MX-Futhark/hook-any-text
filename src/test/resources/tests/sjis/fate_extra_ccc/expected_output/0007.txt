["このあたりが、ほら。
風紀委員として恥ずかしいよね？
でも……"]